### ISSUE TITLE
[BUG] tune_model bug in Voting Ensemble Model

### ISSUE THREAD

**Issue by moezali1** [pycaret/pycaret#1703]
When model object is `pycaret.internal.ensemble._EnsembleForecasterWithVoting` `tune_model` will fail:

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_6868/2247206210.py in <module>
----> 1 tune_model(blender)

~\pycaret\pycaret\internal\utils.py in wrapper(*args, **kwargs)
    779                 if globals_d[name] is None:
    780                     raise ValueError(message)
--> 781             return func(*args, **kwargs)
    782 
    783         return wrapper

~\pycaret\pycaret\time_series.py in tune_model(estimator, fold, round, n_iter, custom_grid, optimize, custom_scorer, search_algorithm, choose_better, fit_kwargs, return_tuner, verbose, tuner_verbose, **kwargs)
    619     """
    620 
--> 621     return _CURRENT_EXPERIMENT.tune_model(
    622         estimator=estimator,
    623         fold=fold,

~\pycaret\pycaret\internal\pycaret_experiment\time_series_experiment.py in tune_model(self, estimator, fold, round, n_iter, custom_grid, optimize, custom_scorer, search_algorithm, choose_better, fit_kwargs, return_tuner, verbose, tuner_verbose, display, **kwargs)
   1883 
   1884         if not param_grid:
-> 1885             raise ValueError(
   1886                 "parameter grid for tuning is empty. If passing custom_grid, "
   1887                 "make sure that it is not empty. If not passing custom_grid, "

ValueError: parameter grid for tuning is empty. If passing custom_grid, make sure that it is not empty. If not passing custom_grid, the passed estimator does not have a built-in tuning grid.

**Comment by Yard1** [comment]
@ngupta23 You can tune the weights - at least that's how the functionality should work.

**Comment by ngupta23** [comment]
@moezali1 @Yard1 - should be implemented in #3534 

```
Also implemented tuning of blended models. The tuning happens using the methods only (not the weights).
```
